features/exploredash/src/main/java/org/dash/wallet/features/exploredash/data/explore/MerchantDao.kt

```diff
@@ -39,6 +39,7 @@ interface MerchantDao : BaseDao<Merchant> {
             AND (:territoryFilter = '' OR territory = :territoryFilter)
             AND (:paymentMethod = '' OR paymentMethod = :paymentMethod)
             AND type IN (:types)
+            AND redeemType <> 'url'
```

Comment:
for all queries we will ignore any redeem url merchants since this is not supported.  This will avoid making a v3 database later on where we need to add a new column.